README.md

```diff
@@ -13,7 +13,7 @@ Please **share** this gem with your friends to get more feedback. I would really
 />](https://rails-db.herokuapp.com/rails/db/)
 
 
-Quick way to inspect your Rails database **http://localhost:3000/rails/db**, see content of tables, export them to CSV, EXPLAIN SQL and run SQL queries.
+Quick way to inspect your Rails database **http://localhost:12345/rails/db**, see content of tables, export them to CSV, EXPLAIN SQL and run SQL queries.
```

Comment:
@ParkinT please note that port 12345 is used only when you running RailsDB as standalone app. If you running it as  engine inside your app port will be standard - 3000. Could you please correct documentation to describe this difference?